pkg/resources/quota/quota.go

```diff
@@ -128,6 +135,20 @@ func GetQuota(quotaParam string, QuotaConfig *corev1.ConfigMap, retQuota *Quota)
 		retQuota.productConfigs[product] = pc
 	}
 
+	platform, err := cluster.GetPlatformType(ctx, c)
```

Comment:
That does sound reasonable, to only call the `GetPlatformType` when we initialise the quota instead of on each Get - I could create a follow up ticket?